doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -78,6 +78,11 @@ nRF Desktop
   * Added Kconfig options that allow erasing dongle bond on the gaming mouse using buttons or config channel.
   * Added two states to enable erasing dongle peer: ``STATE_DONGLE_ERASE_PEER`` and ``STATE_DONGLE_ERASE_ADV``.
 
+* Removed:
```

Comment:
```suggestion
```
Should move to updated section as we are not completely removing the kconfig option.